frontend/src/layouts/footer-layout.tsx

```diff
@@ -0,0 +1,17 @@
+function footer() {
+  const newDate = new Date();
+  const year = newDate.getFullYear();
+
+  return (
+      <footer className="w-full text-center self-end absolute bottom-0 sm:h-[7vh] flex flex-col sm:flex-row items-center justify-between sm:px-4 md:px-[2rem] lg:px-[4rem] text-white bg-[#181818]">
+          <section className="text-[16px]">
```

Comment:
please don't hard code the data, unless there is no other way to solve it. 
1. w.r.t heights use tailwind equivalent of px-2rem something like px-2 or px-1 or px-16 so on
2. use variable for color and define it in tailwind-config.js

Reply:
Correction done!